res/TensorFlowLiteRecipes/DeptwiseConv2D_001/test.recipe

```diff
@@ -0,0 +1,43 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 1 dim: 64 dim: 64 dim: 8 }
+}
+operand {
+  name: "ker"
+  type: FLOAT32
+  shape { dim: 1 dim: 3 dim: 3 dim: 8 }
+}
+operand {
+  name: "bias"
+  type: FLOAT32
+  shape { dim: 8 }
+  filler {
+    tag: "constant"
+    arg: "1.1"
+  }
+}
+operand {
+  name: "ofm"
+  type: FLOAT32
+  shape { dim: 1 dim: 64 dim: 64 dim: 8 }
+}
+operation {
+  type: "DepthwiseConv2D"
+  depthwiseconv2d_options {
+    padding: SAME
+    stride_w: 1
+    stride_h: 1
+    dilation_w_factor: 1
+    dilation_h_factor: 1
```

Comment:
Yes... if this is for dilation test, it will be better to set to `dilation_?_factor != 1`

Reply:
I noticed this because you mentioned it in draft, but it was not fixed here =)